design-documents/asynchronous-import/source-data-retrieving.md

```diff
@@ -0,0 +1,144 @@
+# Asynchronous Import Source data retrieving
+
+## Context
+
+The main idea is to receive data for import from different sources.
+
+## Requirements
+- Retrieving source data is encapsulated in a separate layer (including a separate module);
```

Comment:
Why should it be in a separate module? Can it be used in other functionality than import?